opal/opal/mini.rb

```diff
@@ -14,5 +14,6 @@
 ::Object.require 'corelib/method'
 ::Object.require 'corelib/regexp'
 ::Object.require 'corelib/variables'
-::Object.require 'corelib/io'
 ::Object.require 'opal/regexp_anchors'
+::Object.require 'corelib/io/buffer'
```

Comment:
Should we really need this in opal/mini? Perhaps a better place for it would be opal/full, since not many people use this interface.

Reply:
I put it in mini because io is in mini and i expect io to depend on io/buffer.
But i am still working on io, platform support. I thought using io/buffer in io is a good idea, but i am not 100% sure about this anymore. Depending on further progress with my platform work. Cannot decide yet. Feel free to change, if you prefer, or otherwise wait a bit.

Reply:
Same goes with let's say pattern matching, but pattern matching is rarely used for now, so it takes up valuable asset size. I'll adjust it to `opal/full` and merge.